### ISSUE TITLE
Using short abbreviations to indicate colors, instead of numbers

### ISSUE THREAD

**Issue by douglasblash** [debkbanerji/lego-art-remix#74]
As a better option, instead of using assigned numbers to show the color of each tile, just use a very short abbreviation for the color in each tile, (just like "DA" for dark azure, "MA" for medium azure", "BLB" for "bright light blue", etc, etc...)  This makes it way easier to build!  =)

You might also create the option to choose to use either the BrickLink color names, or the official Lego color names...  That would make it way easier for people buying on BrickLink or on Lego Pick-A-Brick.  Lego has a lot of odd color names when you get into the violet/lilac/purple/pink part of the spectrum!

And you can include a quick reminder key, which just reminds you what the little abbreviations stand for.  The abbreviations could just be the same for everyone.

Here's an example.  It's also possible to just show some of the abbreviations, for the more exotic and subtle colors.

![Earth (Mosaic - True Colors)](https://user-images.githubusercontent.com/102312347/164990219-49b91f01-2a54-4303-9adb-70609553e243.png)

**Comment by debkbanerji** [comment]
I like this idea.  Experimenting with this, but it is a little tricky. Initially, I tried generating abbreviations programatically using the first letter of each word, but there are some edge cases where that doesn't work well (ex: rust vs. red or maersk blue  vs. medium blue). Looks like abbreviations will have to be created manually.

As for the Lego color names, this would be nice, but as you pointed out, is tricky in practice. Once I get the time, I'll poke around to see what mappings from Bricklink to Lego colors exist. Feel free to open another issue for this.

**Comment by douglasblash** [comment]
Yeah, it is a cool idea, and if you just mark this as "planned" then I think you can close issue #44 as superseded...

And if you use three letters, you can do like DTN for dark tan, versus DTQ for dark turquoise, etc.

I can help you with the Lego color names, since I am trying to sort that out myself...

Instead of saying "violet" on the spectrum, they call it "color family lilac", which includes:
medium lilac = it is actually "dark purple"
medium lavender = (same)
lavender = (same)

For the colors that fall partly into the magenta zone, they call that "color family purple", which includes:
bright reddish violet = "magenta"
bright purple = it is actually "dark pink"
light purple = it is actually "bright pink"

**Comment by debkbanerji** [comment]
Yes, we'll need to use 3 letters (some color names are very long). Looking at the code, it'll be more logic than expected to get these to fit in instructions well, but I don't see 2 letters being feasible.
